src/components/AddPayment.vue

```diff
@@ -0,0 +1,227 @@
+<template>
+  <div class="main">
+    <div class="inner">
+      <div class="title-wrapper">
+        <div class="title">
+          <span>支払いの追加</span>
+        </div>
+      </div>
+
+      <div class="input-form-wrapper">
+        <div class="input-payment-name-form">
+          <p>支払い内容</p>
+          <input
+            type="text"
+            id="payment-name"
+            placeholder="例：飛行機代"
+            v-model="inputPaymentTitle"
+          />
+          <span v-bind:class="{ red: inputPaymentTitleError }"
+            >※1文字以上20文字以内でご記入ください</span
+          >
```

Comment:
[should]
ここらへん改行がおかしいような 🤔 
```suggestion
          <span v-bind:class="{ red: inputPaymentTitleError }">※1文字以上20文字以内でご記入ください</span>
```
すべて1行にするでよさそう

Reply:
https://github.com/TakahashiJumpei/seisankun_front_v2/pull/8#discussion_r829548116